AndelaOSP/art-backend
# Add date to Notifications endpoint

#### What does this PR do?
This PR adds a `created_at` field in the incident model.

#### Description of Task to be completed?
we are adding a `created_at` field to show when the incidents were created. we are also ordering the returned output using the created_at field. 

#### How should this be manually tested?
1. On postman, go to the  http://127.0.0.1:8000/api/v1/incidence-reports/. you will see the returned data has a `created_at` field.

#### Any background context you want to provide?
ensure you apply the migrations by running the command, `python manage.py migrate`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2146417/stories/160192851
